pages/en/lb4/Getting-started.md

```diff
@@ -24,28 +24,78 @@ The starting point is [ECMAScript 6](http://www.ecma-international.org/ecma-262/
 - Running the app (node ., npm start)
 " %}
 
+## Installation
+
+Make sure you have the following installed:
+
+- Node.js >= 7.0.0
+- TypeScript >= 2.0.0 `npm i -g typescript`
```

Comment:
Installing packages globally is a bad practice. If we need typescript then it should be a regular or a dev dependency of this project.

Reply:
To add to what Miroslav said, maybe we can tell them to create a new directory, make `node_modules` in it (if `npm i` doesn't do it automatically), and install it locally there for the simple example below, and remove the `npm i -g typescript` from here.

Reply:
I see that Miroslav already touched on that in his comment below, and you've structured it that way in this PR. Nice!